corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -219,6 +219,7 @@
       {% registerurl 'ping_login' %}
       {% registerurl 'ping_session' %}
       {% include 'hqwebapp/includes/inactivity_modal.html' %}
+      {% include 'hqwebapp/includes/new_app_version_modal.html' %}
```

Comment:
Is it intentional that this is inside a `request.session.secure_session` condition, so this feature won't be available to projects not using the shortened inactivity timeout?

Reply:
I also wonder, should this modal only be included in web apps instead of `base.html`?

Reply:
No. It was not intentional. I assumed that it should be along with the inactivity modal logic. 

> I also wonder, should this modal only be included in web apps instead of base.html?

This would make sense.

Reply:
It's worth clarifying with product about the inactivity logic - USH won't care, since all our projects use the inactivity logic, but SaaS probably has an opinion.

Reply:
Talked with Kirti and confirmed that we should be having it for all the users. I have moved the template to cloudcare https://github.com/dimagi/commcare-hq/pull/30778/commits/d1579859abb41260d7dcf457c25d5d0034a726f1

Reply:
The modal will now only show in web apps, but it still won't show for domains that don't have `secure_sessions` on, because they won't have `inactivity_modal.html` or the various initial page data & urls. I think all of [this](https://github.com/dimagi/commcare-hq/blob/d1579859abb41260d7dcf457c25d5d0034a726f1/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L215-L221) needs to be included in `formplayer_home.html`, but only if it hasn't already been included in `base.html` (unless I'm misunderstanding your conversation with Kirti).

Reply:
Thanks, Jenny for pointing it out. I tested on local and you were correct that we need to have the variables in `formplayer_home.html`. I have made the changes to the PR.